type_description_interfaces/README.md

```diff
@@ -0,0 +1,22 @@
+# type_description_interfaces
+
+A package containing message and service definitions for describing and communicating descriptions of other types.
+
+## Messages (.msg)
+
+* [Field](msg/Field.msg): Represents a single field in a ROS Interface Type
+* [FieldType](msg/FieldType.msg): Represents the type of a single field in a ROS Interface Type
+* [IndividualTypeDescription](msg/IndividualTypeDescription.msg): Represents a single ROS Message Type by itself
+* [TypeDescription](msg/TypeDescription.msg): Represents a single ROS Message Type along with all referenced types
+
+## Quality Declaration
+
+This package claims to be in the **Quality Level 1** category, see the [Quality Declaration](QUALITY_DECLARATION.md) for more details.
+
+## TODO
+
+* How to store default values, should store default values? Current storing as string, but will impact hashing.
+* Do we (will we) support fixed sized strings vs. just having bounded strings?
```

Comment:
rosidl_parser does not have a fixed sized string type, only bounded and unbounded. This type isn't exposed to `.msg`, but I guess a user could specify it in `.idl`?

Reply:
Yeah, I think so. Or maybe it was that I think we should have a fixed sized string in the future. I think it's ok, but maybe not ideal, that this file can capture more than what you can put in a message. We'll just give up on either end if a message uses this option and we don't yet support it.